libnavui-base/src/main/java/com/mapbox/navigation/ui/base/internal/utils/MultiOnClickListener.kt

```diff
@@ -0,0 +1,27 @@
+package com.mapbox.navigation.ui.base.internal.utils
+
+import android.view.View
+import java.util.HashSet
+
+class MultiOnClickListener : View.OnClickListener {
```

Comment:
as for me `MultiOnClickListener ` is an object which handles multi-click )
but in our case it's manager over a set of single click listeners.
can we change name?

Reply:
renamed to `ClickListenersDelegate`